Modix/Modules/CoinModule.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Modix.Modules
+{
+    using System;
+    using System.Threading.Tasks;
+    using Discord.Commands;
+    using Modix.Utilities;
+
+    [Group("random"), Name("Coin"), Summary("A bunch of random commands"), ApplyChannelLimits]
+    public class CoinModule : ModuleBase
+    {
+        private readonly Random _r = new Random();
```

Comment:
I'd probably rename `_r` to something a little more descriptive even though it is private. Maybe `_random` would be nice.